[IMP] pos/terminals: update adyen page (#18138)
odoo/documentation: content/applications/finance/payment_providers/adyen.rst

Reviewer: I know you didn't update this but ... isn't it strange just to talk about 'more than 10 million annually' with no unit of measurement/currency? (I notice that on the POS Ayden page you took out this same note, and there it mentioned '$10 million' ... maybe we add the currency symbol here on this page?)

Author: Yes, 💯 ! We can do that! That is a good idea, but I removed it from the POS documentation because I didn't find any reliable information about that. I thought it would be saver to remove the information :/

Reviewer: [Out of PR scope - feel free to ignore]

```suggestion
#. On the :guilabel:`Webhook generation` form, in the :guilabel:`Server configuration` section,
enter your Odoo website URL followed by `/payment/adyen/notification`.
```
Apart from the issue of the guilabel for URL, I'm not sure why we would say 'database URL' here, and 'website URL' in line 35 above. I think we can have the same in both places.

Author: I guess it is because there is a field named "URL" on the Adyen interface, don't you think?

Reviewer: True, probably! But do you think we can still say either 'Odoo _website_ [URL]' or 'Odoo _database_ [URL]' in both places? Just for consistency

Reviewer: See previous comment about this instruction

_#iknowyoudidn'tchangethis_

Reviewer: ```suggestion
#. Go to :menuselection:`Developers --> API URLs`, then copy the :guilabel:`Prefix` and save the value for
the :ref:`payment_providers/adyen/configure_odoo` step.
```

Just to align with other steps ... I guess 'save the value' (rather thank just 'save it') makes sense here too?

Adyen configuration
===================

#. Create an Adyen account if necessary and log into your `Adyen Customer Area

>    <https://authn-live.adyen.com/authn/ui/login?request=eyJBdXRoblJlcXVlc3QiOnsiYWN0aXZpdHlHcm91cCI6IkJPX0NBIiwiY3JlZHNSZWFzb24iOlsiTG9nZ2luZyBpbiB0byBhcHBsaWNhdGlvbiBjYSJdLCJmb3JjZU5ld1Nlc3Npb24iOiJmYWxzZSIsImZvcmdvdFBhc3N3b3JkVXJsIjoiaHR0cHM6XC9cL2NhLWxpdmUuYWR5ZW4uY29tXC9jYVwvbG9iYnlcL3Bhc3N3b3JkLXJlc2V0XC9mb3Jnb3QtcGFzc3dvcmQiLCJyZXF1ZXN0VGltZSI6IjIwMjYtMDUtMjJUMTM6MTQ6MzgrMDI6MDAiLCJyZXF1ZXN0ZWRDcmVkZW50aWFscyI6W3siUmVxdWVzdGVkQ3JlZGVudGlhbCI6eyJhY2NlcHRlZEFjdGl2aXR5IjpbeyJBY2NlcHRlZEFjdGl2aXR5Ijp7ImFjdGl2aXR5R3JvdXAiOiJCT19DQSIsImFjdGl2aXR5VHlwZSI6IklNUExJQ0lUIiwibWlsbGlzZWNvbmRzQWdvIjo5MDAwMDB9fV0sInR5cGUiOiJQQVNTV09SRCJ9fSx7IlJlcXVlc3RlZENyZWRlbnRpYWwiOnsiYWNjZXB0ZWRBY3Rpdml0eSI6W3siQWNjZXB0ZWRBY3Rpdml0eSI6eyJhY3Rpdml0eUdyb3VwIjoiQk9fQ0EiLCJhY3Rpdml0eVR5cGUiOiJHUkFDRV9DT09LSUUiLCJtaWxsaXNlY29uZHNBZ28iOjB9fV0sInR5cGUiOiJUV09fRkFDVE9SIn19XSwicmVxdWVzdGluZ0FwcCI6ImNhIiwicmV0dXJuVXJsIjoiaHR0cHM6XC9cL2NhLWxpdmUuYWR5ZW4uY29tXC9jYVwvY2FcLyIsInNpZ25hdHVyZSI6IlYwMDRTeEtQRlMyNzZMWDV5N01aSGx3SHhHMFRSZHc2aG9GUHZvUHhjOXpSSktQTT0ifX0%3D>`_.

Reviewer: ```suggestion
   <https://ca-live.adyen.com/>`_.
```
[I asked Gemini's help for this]
This url brings you to the same login screen and feels a lot less risky than keeping that long request ID.

To update everywhere.

Author: Great idea ❤️

   to enable the **Multiple partial capture** feature.
#. In the Adyen Customer Area, go to :menuselection:`Developers --> API credentials` and click the
   relevant API credential user name in the list or click :icon:`fa-plus-circle` :guilabel:`Create
   new credential` to create a new one.
#. In the :guilabel:`Server settings` section, click :guilabel:`Generate API key`, then click the
   :icon:`fa-copy` (:guilabel:`Copy API key`) icon and save the value for the
   :ref:`payment_providers/adyen/configure_odoo` step.
#. In the :guilabel:`Client settings` section, click :guilabel:`Generate client key`, then click the
   :icon:`fa-copy` (:guilabel:`Copy client key`) icon and save the value for the
   :ref:`payment_providers/adyen/configure_odoo` step.
#. Enter your Odoo website URL in the :guilabel:`Add allowed origins` field, then click
   :guilabel:`Add`.
#. Click :guilabel:`Save changes` at the bottom of the page.
#. Go to :menuselection:`Developers --> Webhooks` in the left menu and click :icon:`fa-plus-circle`
   :guilabel:`Create new webhook`.
#. In the :guilabel:`Create new webhook` popup, click :guilabel:`Add` on to the :guilabel:`Standard

>    webhook` line.

Reviewer: ```suggestion
#. In the :guilabel:`Create new webhook` popup, click :guilabel:`Add` on the :guilabel:`Standard
  webhook` line.
```

==================

> #. :ref:`Navigate to the Adyen payment provider<payment_providers/add_new>`.

Reviewer: _#iknowyoudidn'tupdatethis_

On the other payment provider pages we switch the wording a little and link to the 'Supported providers' section, which is better I think:
          
```suggestion
#. :ref:`Navigate to the payment provider Ayden <payment_providers/supported_providers>`.
```

> #. Fill in the :ref:`Merchant account <adyen/merchant-account>`, :guilabel:`API Key`,

Reviewer: ```suggestion
#. Fill in the :ref:`Merchant Account <adyen/merchant-account>`, :guilabel:`API Key`,
```
